### ISSUE TITLE
[Feature Request]: Add an option to remove a profile picture after it has been set.

### ISSUE THREAD

**Issue by Vishwajith-Shettigar** [oppia/oppia-android#5188]
### Is your feature request related to a problem? Please describe.

There is no option to remove a profile picture. When I set a profile picture, I have only two options available:
1.View profile picture.
2.Choose from the library (which allows me to update the profile picture).

### Describe the solution you'd like

Once I upload my profile picture, there should be an option to remove the profile picture.

**Comment by Vishwajith-Shettigar** [comment]
@adhiamboperes and @seanlip, could you please share your point of view about this feature request.

**Comment by Tejas-67** [comment]
Hey @Vishwajith-Shettigar, I would like to work on this issue.

**Comment by adhiamboperes** [comment]
> Hey @Vishwajith-Shettigar, I would like to work on this issue.

@Tejas-67 please work on this issue instead https://github.com/oppia/oppia-android/issues/4760

**Comment by Tejas-67** [comment]
@adhiamboperes I am working on #4760

**Comment by ShubhadeepKarmakar** [comment]
@adhiamboperes [Block on Product] means?

**Comment by adhiamboperes** [comment]
> @adhiamboperes [Block on Product] means?


Needs product approval for implementation.

**Comment by ShubhadeepKarmakar** [comment]
Like this....?

https://github.com/oppia/oppia-android/assets/99060332/f437e34a-36ae-487f-81d0-28cbf2ba031f

**Comment by seanlip** [comment]
I think it's fine to have this feature but we should get some mocks from the design team for it so that it is consistent with the rest of the app. I'll file an issue on their repo, but let's pause development on this for now until they've done the mocks. Thanks.

**Comment by ShubhadeepKarmakar** [comment]
> I think it's fine to have this feature but we should get some mocks from the design team for it so that it is consistent with the rest of the app. I'll file an issue on their repo, but let's pause development on this for now until they've done the mocks. Thanks.

Ok.

**Comment by Vishwajith-Shettigar** [comment]
> Like this....?
> 
>  delete.button.mp4

Yes, looks pretty good. But instead of leaving view blank, should display Image placeholder.

**Comment by theayushyadav11** [comment]
Hey @adhiamboperes ,can you plz assign this issue to me?

**Comment by BenHenning** [comment]
> I think it's fine to have this feature but we should get some mocks from the design team for it so that it is consistent with the rest of the app. I'll file an issue on their repo, but let's pause development on this for now until they've done the mocks. Thanks.

Do you still feel strongly about this @seanlip? I feel like the proposal in the video above might be reasonable since it's using an existing dialog window. We should probably tweak the wording to be 'Unselect Photo' or something to not make the user think it's actually being deleted, but otherwise it seems reasonable to me.

**Comment by BenHenning** [comment]
Just to follow up: I think it's fine to proceed without UX's input here, though we can FYI them on a PR.

This is a really nice small project for someone to work on, though it's not quite "starter issue" material.
